pypopquiz/io.py

```diff
@@ -156,6 +217,7 @@ def verify_input(input_data: Dict) -> None:
         }
     }
     jsonschema.validate(input_data, schema)
```

Comment:
You didn't change anything in the JSON schema? New things are now possible, right? Although optional, you can still describe their expected types and sub-types.